plugins/modules/packaging/language/pear.py

```diff
@@ -166,9 +225,12 @@ def install_packages(module, state, packages):
         if state == 'latest':
             command = 'upgrade'
 
-        cmd = "%s %s %s" % (_get_pear_path(module), command, package)
-        rc, stdout, stderr = module.run_command(cmd, check_rc=False)
+        current_prompt_regex = (None, None)
+        if has_prompt and (len(prompts) > 0 and i < len(prompts)):
+            current_prompt_regex = prompts[i]
 
+        cmd = "%s %s %s" % (_get_pear_path(module), command, package)
+        rc, stdout, stderr = module.run_command(cmd, check_rc=False, prompt_regex=current_prompt_regex[0], data=current_prompt_regex[1])
```

Comment:
Yeah I saw this and I need to set binary_data to True I think because module_run_command() considers I pass a None _data_ variable even tho it's the string is only empty so I think I need to set "\n" manually to prevent such behavior